src/chat-model.ts

```diff
@@ -646,7 +646,45 @@ ${toolsList}
           const cellType = cell.cell_type;
           const lang = cellType === 'code' ? kernelLang : cellType;
 
-          return `**Cell [${cellInfo.id}] (${cellType}):**\n\`\`\`${lang}\n${code}\n\`\`\``;
+          let outputs = '';
+          if (cellType === 'code' && Array.isArray(cell.outputs)) {
+            outputs = cell.outputs
+              .map(
+                (output: {
+                  output_type: string;
+                  text: any;
+                  traceback: any[];
+                  data: any;
+                }) => {
+                  if (output.output_type === 'stream') {
+                    return output.text;
+                  } else if (output.output_type === 'error') {
+                    return output.traceback.join('\n');
+                  } else if (
+                    output.output_type === 'execute_result' ||
+                    output.output_type === 'display_data'
```

Comment:
I wonder if there are use cases for it. Testing it with `ipywidgets`, the output is truncated very early:
For an `Image` widget for example, the output is `Image(value=b'\xff\xd8\xff\xe1\x15\xe6Exif\x00\x00MM\x00*\x00\x00\x00\x08\x00\x10\x01\x00\x00\x03\x00\x00\x00\…`